spec/migrations/20131107000917_expand_dialog_field_default_value_size_spec.rb

```diff
@@ -1,5 +1,5 @@
 require "spec_helper"
-require Rails.root.join("db/migrate/20131107000917_expand_dialog_field_default_value_size.rb")
+require_migration
```

Comment:
@kbrock can you verify we don't use other "ways" to require the migration in specs?  I don't know that we do Rails.root.join... on all of them...